src/modules/twap/state/twapOrdersListAtom.ts

```diff
@@ -14,7 +14,7 @@ import { updateTwapOrdersList } from '../utils/updateTwapOrdersList'
 
 export type TwapOrdersList = { [key: string]: TwapOrderItem }
 
-export const twapOrdersAtom = atomWithStorage<TwapOrdersList>('twap-orders-list:v5', {})
+export const twapOrdersAtom = atomWithStorage<TwapOrdersList>('twap-orders-list:v6', {})
```

Comment:
We will revert to `v0` before release, right?

![](https://i.imgflip.com/5e12dg.jpg)

Reply:
Hahaha, yep, will reset to 1

Reply:
Done